resolvconf/resolvconf.go

```diff
@@ -223,8 +223,20 @@ func GetOptions(resolvConf []byte) []string {
 // Build writes a configuration file to path containing a "nameserver" entry
 // for every element in dns, a "search" entry for every element in
 // dnsSearch, and an "options" entry for every element in dnsOptions.
-func Build(path string, dns, dnsSearch, dnsOptions []string) (*File, error) {
+func Build(path string, dns, dnsSearch, dnsOptions []string, comments string) (*File, error) {
```

Comment:
if there is multiple comments maybe []strings is more appropriate instead of doing the split on \n below